### PR TITLE
Add prayer bar color configuration options

### PR DESCRIPTION
**Pull request by nfiniteecho**
Adds options to #3879 to configure fill and background color of prayer bar.  

GIF

Add new default colors of cyan (fill) and black (background) for ease of visibility. The old default colors of cyan and white were hard to see with high brightness and with colorblindness.

### CODE REVIEW

**Comment by deathbeam** [comment]
The prayer was was not made configurable on purpose, because its showing prayer and it should have specific color. But I like the black background with cyan here as default more, looks better.

**Comment by nfiniteecho** [comment]
@deathbeam Yeah that makes sense generally to make it one color, but I think color customization options would be nice to have for those that may want them.  Prayer doesn't really have one defined color: it's meter is blue in the status orb, green potion for prayer pots, and a pink potion for super restores.

Not the biggest deal to leave out these customization options because I think a default black/cyan color scheme will fix most visibility problems.  What do you think?

**Comment by deathbeam** [comment]
Sounds good.

File: runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerConfig.java [hunk 1]
```diff
@@ -32,87 +32,87 @@
 public interface PrayerConfig extends Config
 {
 	@ConfigItem(
-		position = 0,
-		keyName = "prayerflick",
-		name = "Prayer flick helper",
-		description = "Enable the prayer flick helper"
+			position = 0,
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerConfig.java]: Do not reindent the fields here, see https://github.com/runelite/runelite/wiki/Code-Conventions

**Comment by Nightfirecat** [comment]
This was added in #5635.

**Pull request closed without merging**